scripts/bring_assisted_service.sh

```diff
@@ -18,14 +18,17 @@ export REPO_NAME=${REPO_NAME:-assisted-service}
 export JOB_TYPE=${JOB_TYPE:-}
 export PULL_BASE_REF=${PULL_BASE_REF:-master}
 
+
 if ! [[ -d "assisted-service" ]]; then
   echo "Can't find assisted-service source locally, cloning ${SERVICE_REPO}"
   retry -- git clone "${SERVICE_REPO}"
+elif
+  [[ ${SERVICE_REPO} != $(cd assisted-service && git remote get-url origin) ]]; then
+  echo "assisted-service repository found locally but with a different origin, replacing with ${SERVICE_REPO}"
+  rm -rf ./assisted-service
+  retry -- git clone "${SERVICE_REPO}"
```

Comment:
What happens if as a developer, I want to test with my own branch of assisted service? I don't think we want to align in with test-infra :thinking:

Reply:
You can still test your own branch as I specified in the doc